provider/k8s/app.go

```diff
@@ -52,6 +52,19 @@ func (p *Provider) AppCreate(name string, opts structs.AppCreateOptions) (*struc
 		return nil, errors.WithStack(err)
 	}
 
+	sourceSecret, _ := p.Cluster.CoreV1().Secrets(p.RackName+"-system").Get(context.TODO(), "registries", am.GetOptions{})
```

Comment:
why are you ignoring error here? you should not ignore error

also use `p.Namespace` for system namespace